homeassistant/components/knx/switch.py

```diff
@@ -21,17 +33,18 @@ class KNXSwitch(KnxEntity, SwitchEntity):
 
     def __init__(self, device: XknxSwitch):
         """Initialize of KNX switch."""
+        self._device: XknxSwitch
         super().__init__(device)
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return true if device is on."""
-        return self._device.state
+        return bool(self._device.state)
```

Comment:
This is Optional[bool] in xknx. None on init until the first telegram sets a state. ToggleEntity doesn't support this.